src/commands/bindings.rs

```diff
@@ -87,7 +91,55 @@ The expected file is {wit_path:?}"#
                 println!("Bindings generated for Rust in {0}. You need to add the `wit-bindgen` crate to your Rust Spin app - e.g., `cargo add wit-bindgen`", self.output.to_str().expect("Failed to parse output path"));
             }
             BindingsLanguage::Ts => {
-                todo!("generate ts")
+                let imported_interfaces = get_imported_interfaces(&resolve, world_id);
```

Comment:
This would seem a good opportunity to pull both this and the Rust arm out into helper functions, so that readers can follow the top-to-bottom flow of `run` without needing to track where they are entering and existing lengthy case blocks.